.gitignore

```diff
@@ -13,5 +13,5 @@ npm-debug.log
 .project
 test/memory.json
 .nyc_output
-
+dist
```

Comment:
Please add `dist` to [.npmignore](https://github.com/strongloop/loopback-datasource-juggler/blob/master/.npmignore) too.